cmd/kubelet/app/options/options.go

```diff
@@ -114,6 +112,7 @@ func NewKubeletServer() *KubeletServer {
 			RktPath:                        "",
 			RktStage1Image:                 "",
 			RootDirectory:                  defaultRootDir,
+			RuntimeContainer:               "/docker-daemon",
```

Comment:
If you can review my PR on Monday, maybe we can find a suitable middle ground but I don't like exposing yet another container name for users to manage.